test/swingsetTests/test-contractHost.js

```diff
@@ -0,0 +1,179 @@
+import { test } from 'tape-promise/tape';
```

Comment:
I think these test-*.js files should be in the same directory as the vats they rely on. There doesn't seem to be a good reason to separate them from the code they rely on.

So this file would be test/swingsetTests/scaffolding/contractHost/ along with bootstrap and vat-alice, vat-bob, and vat-fred.

Reply:
Sounds good!